env_support/cmsis-pack/lv_conf_cmsis.h

```diff
@@ -61,6 +61,16 @@
  *(Not so important, you can adjust it to modify default sizes and spaces)*/
 #define LV_DPI_DEF 130     /*[px/inch]*/
 
+/*customize tick-get */
+#if defined(__PERF_COUNTER__) && __PERF_COUNTER__
+    #define LV_GLOBAL_INIT(__GLOBAL_PTR)                                    \
```

Comment:
It makes sense to me.
@bjsylvia please take a look as well.

Reply:
Yes, with this method, the support for ESP timer and Arduino timer can be added back.